src/enclave/Common/mCrypto.h

```diff
@@ -0,0 +1,81 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#ifndef OE_SAMPLES_ATTESTATION_ENC_CRYPTO_H
+#define OE_SAMPLES_ATTESTATION_ENC_CRYPTO_H
```

Comment:
Since this is not our code but I assume comes from the OpenEnclave sample code, perhaps we could put a link to the original in a comment somewhere?

Also, I wonder why the filename is "mCrypto.h" rather than "Crypto.h"?

Reply:
Added. "mCrypto" is used to distinguish this from the "Crypto.h" under src/enclave/Enclave. The "m" here refers to the fact that the library is based on mbedtls instead of our internal enclave crypto implementation.